hazelcast-jet-core/src/main/java/com/hazelcast/jet/pipeline/JdbcSinkBuilder.java

```diff
@@ -128,13 +139,32 @@
         return this;
     }
 
+    /**
+     * Sets the batch size limit for the sink. If the JDBC driver supports
+     * batched updates, this defines the limit at which batches will be
+     * executed and then re-filled while processing arriving items.
+     * <p>
+     * The default batch size limit is {@value DEFAULT_BATCH_LIMIT}.
+     *
+     * @param batchLimit the batch size limit for the sink
+     * @return this instance for fluent API
+     * @since 4.5
+     */
+    @Nonnull
+    public JdbcSinkBuilder<T> batchLimit(int batchLimit) {
+        this.batchLimit = batchLimit;
```

Comment:
can we check that if batchLimit is a positive number.
You can use `com.hazelcast.internal.util.Preconditions#checkPositive`

Reply:
Thanks for the suggestion. Added it now.
Did it here and also in all other places where `batchLimit` should be checked as well.